src/services/AttachmentResolver.js

```diff
@@ -43,42 +47,77 @@ export default class ImageResolver {
 	/*
 	 * Resolve a given src.
 	 * @param { string } the original src in the node.
-	 * @returns { Array<string> } - resolved urls to try.
+	 * @returns { Array<Array> } - resolved candidates to try.
```

Comment:
This returns an Array of Objects, right?

Reply:
Oops yes, fixed.